### PR TITLE
First Commit, HTML and CSS files written

### PR DESCRIPTION
**Pull request by magnanimus1994**
This is a pull request. I believe I know what that is.

### CODE REVIEW

**Comment by candre717** [comment]
LGTM

File: christina.html [hunk 1]
```diff
@@ -1,96 +0,0 @@
-<html>
```
Comment (chdastolfo) [inline christina.html]: NOOOOO WHY IS IT GONEEEEEE.

File: css/christina.css [hunk 2]
```diff
@@ -1,51 +0,0 @@
-body{
-    background-image: url('../img/background.jpg');
```
Comment (chdastolfo) [inline css/christina.css]: SAAAAAME I WANT MY CSS.

File: index.html [hunk 3]
```diff
@@ -1,35 +0,0 @@
-<html>
```
Comment (chdastolfo) [inline index.html]: COME BACK INDEX.HTML

**Comment by chdastolfo** [comment]
This looks good except for my files being gone on your branch :(. They'll have to go back in before we can merge this, or it will remove those files from the master branch as well.

**Comment by magnanimus1994** [comment]
Okay I believe I added back all the files.

**Pull request closed without merging**